paper-drawer-panel.html

```diff
@@ -134,19 +134,19 @@
 
     'use strict';
 
-   // this is the only `paper-drawer-panel` in
+   // this would be the only `paper-drawer-panel` in
```

Comment:
@notwaldorf haha, I was thinking that the sentence should be in future rather than present tense.